rocAL/rocAL_pybind/rocal_pybind.cpp

```diff
@@ -291,7 +291,7 @@ namespace rocal{
     }
 
     */
```

Comment:
Can we remove these commented lines?

Reply:
Removed the commented lines from the file